lib/src/golden_test_adapter.dart

```diff
@@ -383,15 +382,11 @@ class FlutterGoldenTestWrapper extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MediaQuery(
-      data: MediaQuery.maybeOf(context) ??
-          MediaQueryData.fromWindow(WidgetsBinding.instance.window),
-      child: _LocalizationWrapper(
-        child: Theme(
-          data: _resolveThemeOf(context),
-          child: _NavigatorWrapper(
-            child: child,
-          ),
+    return _LocalizationWrapper(
```

Comment:
@jeroen-meijer Do you think this will cause any issues removing the `MediaQuery`? I don't remember why it was written this way, and maybe we should keep it but use the view from the platformDispatcher, ie something like:
```dart
data: MediaQuery.maybeOf(context) ??
          MediaQueryData.fromView(
            WidgetsBinding.instance.platformDispatcher.implicitView ??
                View.of(context),
          ),
```

Reply:
Just adding some more context here as well:

Another option would be to adjust the Test that checks for the media query to ensure all of them are the correct dimensions (adjustment would be needed as the `findsOneWidget` is not satisfied anymore)
However I'm not sure how to hit the `??` line for 100% Test Coverage here as the media query from `maybeOf` is always used